libs/tree-sitter-wing/grammar.js

```diff
@@ -108,9 +108,9 @@ module.exports = grammar({
       seq(
         "struct",
         field("name", $.identifier),
-        optional(seq("extends", commaSep($.identifier))),
+        optional(seq("extends", commaSep(field("extend", $.identifier)))),
```

Comment:
nit: I feel like this should be plural, both to match the keyword and also to better signify that there can be many of them.

```suggestion
        optional(seq("extends", commaSep(field("extends", $.identifier)))),
```